src/rpc/simulate/compressor/LrAmCompressor.h

```diff
@@ -148,9 +148,10 @@ namespace Rpc
       RFieldDft<D> residK_;
      
       /**
-      * IntraCorrelation.
+      * IntraCorrelation in fourier space 
+      * Calculated by IntraCorrlation class.
       */
-      RField<D> intraCorrelation_;
+      RField<D> intraCorrelationK_;
```

Comment:
This seems strange to me - Why would you explicitly say that something is an intra correlation in Fourier space (also indicated by the name intraCorrelationK_, but store it a RField<D> object, which is designed for real space fields? We have a RFieldDft<D> class designed to store the discrete Fourier transform of a real field, which stores complex numbers in a smaller memory footprint to take into account the complex conjugate relationships between DFT values for k and -k for a real field by only storing one of them.